### ISSUE TITLE
Potential Maintainer

### ISSUE THREAD

**Issue by kylemasc917** [ubsuny/CP1-24-HW2#7]
One of the first things that needs to be sorted is the maintainer for this project. This should be finalized no later than the end of day Friday potentially after lecture. I will leave this open for discussion until lecture on Friday So that those interested parties may discuss it here. It should be someone with some experience of git and github but not as much experience with team organization and team communication so this can be a new learning experience.

**Comment by kylemasc917** [comment]
I am open to being the maintainer for this project long as no one else wants to do it instead.

**Comment by dnxjay** [comment]
Agreed this is something that should be done first, you have my vote to appoint you for this

**Comment by avgagliardo** [comment]
> Agreed this is something that should be done first, you have my vote to appoint you for this

likewise

**Comment by laserlab** [comment]
looks enough votes to me :-) I promote @kylemasc917 to maintainer.

**Comment by HeshamElsaman** [comment]
Since our maintainer has been chosen, this should close this issue.
@kylemasc917, please, close it if it's no longer needed.

**Comment by dhamalakamal** [comment]
I just tried if any other person other than who raised an issue can close the issue or not. So, I reopened it back again.

**Comment by kylemasc917** [comment]
I will close this issue now.
